src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -136,6 +136,9 @@
     <Compile Remove="$(PathToMsalSources)\Platforms\**\*.*;$(PathToMsalSources)\Resources\*.cs" />
     <Compile Remove="$(PathToMsalSources)\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs" />
     <EmbeddedResource Include="$(PathToMsalSources)\Properties\Microsoft.Identity.Client.rd.xml" />
+    <Compile Include="IntuneAppProtectionPolicyRequiredException.cs">
```

Comment:
VS has some problems understanding our project structure, so it manually adds stuff here. This is wrong, please revert this chage.